backend/cmd/headlamp.go

```diff
@@ -212,6 +287,19 @@ func GetDefaultKubeConfigPath() string {
 	return filepath.Join(os.Getenv("HOME"), ".kube", "config")
 }
 
+func getClusterAuthType(w http.ResponseWriter, r *http.Request) {
```

Comment:
Should we instead include the cluster auth types together with the cluster config we already have an endpoint for?

Reply:
yes we should do that, it would reduce one extra api call to fetch the auth type